src/Razor/test/Microsoft.AspNetCore.Razor.VSCode.Test/yarn.lock

```diff
@@ -2149,12 +2149,8 @@ micromatch@^4.0.4:
     picomatch "^2.2.3"
 
 "microsoft.aspnetcore.razor.vscode@link:../../src/Microsoft.AspNetCore.Razor.VSCode":
-  version "0.0.1"
-  dependencies:
-    ps-list "^7.0.0"
-    vscode-html-languageservice "^4.2.1"
-    vscode-languageclient "5.2.1"
-    vscode-languageserver-textdocument "^1.0.3"
+  version "0.0.0"
+  uid ""
```

Comment:
This looks bad

Reply:
I suspect that this is because our vscode build is currently broken, but I've not dealt with yarn.lock files muck. @allisonchou might know something about the state of that package?

Reply:
You're seeing the first time I've touched a yarn.lock file, so I'm none the wiser, but I think I'll revert this bit just in case